docs/querying/sql.md

```diff
@@ -185,20 +182,17 @@ UNION ALL
 SELECT COUNT(*) FROM tbl WHERE my_column = 'value2'
 ```
 
-With top-level UNION ALL, no further processing can be done after the UNION ALL. For example, the results of the
-UNION ALL cannot have GROUP BY, ORDER BY, or any other operators applied to them.
+> With top-level queries, you can't apply GROUP BY, ORDER BY, or any other operator to the results of a UNION ALL.
```

Comment:
I wonder if the "no further processing" is is necessary. It does get lost in the revision.

Reply:
I think it's covered by "to the results".

Reply:
Updated doc with Ben's suggestions.